IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -39,6 +39,7 @@
 #import "MSIDAccountMetadataCacheAccessor.h"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->